app/models/concerns/users/organization_manipulator.rb

```diff
@@ -25,15 +25,17 @@ def can_update_organization_county?
     def create_organization(params)
       raise PermissionError unless can_create_organization?
       org_params = params.require(:organization)
-      Organization.create! org_params.permit(:name, :address, :county, :phone_number, :email)
+      org_params[:addresses_attributes].select! { |_, h| h[:address].present? }
+      Organization.create! org_params.permit(:name, :phone_number, :email, addresses_attributes: [:address, :id])
     end
 
-    def update_organization(params)
+    def update_organization(params) # rubocop:disable Metrics/AbcSize
```

Comment:
In this kind of situation I've opted to break the method into a few different methods, like `organization.update_details(params)` and then `organization.update_addresses(params)` to both avoid needing to disable rubocop and break up the methods into smaller, single purpose methods.